.circleci/config.yml

```diff
@@ -10,21 +10,33 @@ jobs:
   build:
     executor: golang
 
+    parameters:
+      version:
+        type: string
+
     steps:
     - checkout
     - setup_remote_docker
     - run: docker info
-    - run: make
+    - run: make VARIANTS=base VERSION=<< parameters.version >>
```

Comment:
```suggestion
    - run: make VERSION=<< parameters.version >>
```

Reply:
I'm doing this intentionaly to split base and main into two steps.